src/ExportMethods.targets

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), build.tasks))\build.tasks" />
+  <PropertyGroup>
+    <ExportPath>$(ProjectRoot)bin\$(Configuration)\Exports\</ExportPath>
+    <LibName>$(MSBuildProjectName.Replace('.Desktop', ''))</LibName>
+  </PropertyGroup>
+  <Target Name="AfterBuild">
```

Comment:
There can only be one AfterBuild target. So I try to avoid ever using it. Instead, give the Target a meaningful name (e.g. "CreatePInvokeExportsFile") and add an `AfterTargets="Build"` attribute to the `Target` tag as well.